Directory.Build.props

```diff
@@ -1,4 +1,4 @@
-<Project TreatAsLocalProperty="RepoRoot">
+<Project>
```

Comment:
side note: just realized that probably suitable place for BuildArchitecture defined in this file is CalculateArch:
https://github.com/dotnet/runtime/blob/96c97c2cf48a4dc2bd9365c67f442b68514a007c/eng/Configurations.props#L109-L110
after line 110:
```xml
<!-- Determine what architecture we are building on. -->
<BuildArchitecture Condition="'$(BuildArchitecture)' == ''">$(_hostArch)</BuildArchitecture>
```

Reply:
Thanks :) Also side note, is it intentional that `BuildArchitecture` is passed in on Unix always and makes the mentioned code line actually a no-op? https://github.com/dotnet/runtime/blob/96c97c2cf48a4dc2bd9365c67f442b68514a007c/eng/build.sh#L462

Reply:
OK I just saw that the property is in most cases passed in as an empty string which I presume msbuild can then later override.